09/bst-height.instructor.md

```diff
@@ -0,0 +1,40 @@
+# Instructor: BST Height
```

Comment:
Should we include space complexity for instructors since students will likely want to discuss? 

If we do, should probably include that tree is not necessarily balanced for this problem. I believe that will affect the space complexity but not time complexity.